solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -678,20 +700,28 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
         uint256 _targetAmount)
         internal view returns (uint256)
     {
-        // conversions to the primary reserve apply normal fees
-        if (_targetToken == primaryReserveToken) {
-            return super.calculateFee(_targetAmount);
+        uint256 fee;

```

Comment:
I think the function should return super.calculateConversionFee + the adjusted fee

Reply:
Fixed.